Remove kubeless and lambda mentions in docs (#8144)
kyma-project/kyma: docs/compass/08-01-basic-flow-tutorial.md

1. Go back to the Kyma Console UI. You can see that the `test-app` Application is registered in the **Applications** view. Select `test-app` and bind it to your Namespace by selecting the **Create Binding** button.

> 2. From the drop-down list in the top-right corner, select your Namespace and go to the **Catalog** view. You will see your services available under the **Services** tab. Provision the Service Instance by choosing your Package and clicking the **Add** button in the top-right corner of the page.

Reviewer: ```suggestion
2. From the drop-down list in the top-right corner, select your Namespace and go to the **Catalog** view. You will see your services available under the **Services** tab. Provision the ServiceInstance by choosing your Package and clicking the **Add** button in the top-right corner of the page.
```

I did mean that lack of space in `ServiceInstance` 😄 Please see the [guidelines](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/08-style-and-terminology.md#capitalized-terms).

3. Create a function. In the **Overview** tab, click the **Deploy new resource** button and upload the file with the [function](./assets/function.yaml).

4. Expose your function following [this](/components/serverless/#tutorials-expose-a-function-with-an-api-rule) tutorial. Follow the instructions in the `Console UI` tab.

> 5. Bind the Service Instance to your function following [this](/components/serverless/#tutorials-bind-a-service-instance-to-a-function) tutorial. Follow the instructions in the `Console UI` tab.

Reviewer: ```suggestion
5. Bind the ServiceInstance to your function following [this](/components/serverless/#tutorials-bind-a-service-instance-to-a-function) tutorial. Follow the instructions in the `Console UI` tab.
```

I did mean that lack of space in `ServiceInstance` 😄 Please see the [guidelines](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/08-style-and-terminology.md#capitalized-terms).

> 2. From the drop-down list in the top-right corner, select your Namespace and go to the **Catalog** view. You will see your services available under the **Services** tab. Provision the service instance by choosing your Package and clicking the **Add** button in the top-right corner of the page.

Author: In other tutorials we have CR written separately and so I prefer to leave the name separated :)

3. Create a function. In the **Overview** tab, click the **Deploy new resource** button and upload the file with the [function](./assets/function.yaml).

4. Expose your function by [this](/components/serverless/#tutorials-expose-a-function-with-an-api-rule) tutorial. Follow the instructions in the `Console UI` tab.

5. Bind to your function ServiceInstance by [this](/components/serverless/#tutorials-bind-a-service-instance-to-a-function) tutorial. Follow the instructions in the `Console UI` tab.

### Cleanup

Clean up your cluster after going through this tutorial. To do so, delete your resources in the following order:

> 1. Go to the **Functions** tab and delete `call-order-service` function.

Reviewer: ```suggestion
1. Go to the **Functions** tab and delete the `call-order-service` function.
```
